### PR TITLE
docs: fix typo and make comment clearer

### PR DESCRIPTION
**Pull request by paarthmadan**
Resolves #ISSUE_NUMBER

----

### Before the change?
- Just a quick doc change, thought this comment was a little bit hard to parse

### After the change?

* 

### Pull request checklist
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)

### Does this introduce a breaking change?

Please see our docs on [breaking changes](https://github.com/octokit/.github/blob/master/community/breaking_changes.md) to help!

- [ ] Yes
- [ ] No

----

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -590,7 +590,7 @@ try {
   // your code here that sends at least one Octokit request
   await octokit.request("GET /");
 } catch (error) {
-  // Octokit errors always have a `error.status` property which is the http response code nad it's instance of RequestError
+  // Octokit errors are instances of RequestError, so they always have an `error.status` property for the HTTP response code.
```
Comment (wolfy1339) [inline README.md]:
```suggestion
  // Octokit errors are instances of RequestError, so they always have an `error.status` property containing the HTTP response code.
```
Maybe this is better

Comment (paarthmadan) [reply README.md]: Yep, makes sense to me!

**Review by wolfy1339 — APPROVED** [review]
